src/components/fields/Dropdown.js

```diff
@@ -11,11 +11,25 @@ export class UnconnectedDropdown extends Component {
       placeholder = this.props.fullValue;
     }
 
+    let options = this.props.options;
+
+    if (
+      this.props.attr === 'geo.projection.type' &&
```

Comment:
Is there any way we can do this outside of the `Dropdown`? I'd rather not bake in this super-specific attribute stuff. Can we create a `ProjectionsDropdown` instead?